nixos/modules/services/desktops/geoclue2.nix

```diff
@@ -35,23 +85,98 @@ in
         '';
       };
 
+      enableNmea = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Whether to fetch location from NMEA sources on local network.
+        '';
+      };
+
+      extraWhitelistedAgents = mkOption {
+        type = types.listOf types.str;
+        default =  [ "" ];
```

Comment:
I think this should be `[]` by default.

Reply:
`[]` and `[ "" ]` are both lists of strings, but I don't see a reason to include `""` by default

Reply:
I agree, I think this will result in the `whitelist` field always containing an empty string due to the merge strategy.